### DISCUSSION TITLE
Bar Chart: how to draw a minimum height for series data value 0?

### DISCUSSION THREAD

**Discussion by yepMad** [apexcharts/apexcharts.js#3166] [Q&A]
I have a chart with 7 bars. I have no labels and no grid. I would like that when a value is 0 the bar is drawn in minimum size, but still exists.

Current state:
![image](https://user-images.githubusercontent.com/22564368/169904637-afbe60f3-3e0f-47ea-aa18-ed4ff80ae867.png)

How is it when there is data for the 7 bars:
![image](https://user-images.githubusercontent.com/22564368/169904691-19cee47d-3cfc-4d2d-93db-e18d500f90b8.png)

**Comment by Tundrawlkr** [comment]
I am pretty new to this project but I can say that if you set the main chart type to 'area' and then set your series type to 'bar' you'll get a rendered small bar even if the value is 0.  I don't know if it's the intended functionality or not but it will do that.

Edit: here's a screenshot of the getting started Codepen from the apexcharts website with what I mentioned above implemented

![image](https://user-images.githubusercontent.com/73982296/173151076-3253c241-c781-49a6-8770-8463064d016c.png)

**Comment by tedeh** [comment]
For a "sparklines-type" chart I've been successful just incrementing every data point by 1, and then changing the formatter to decrement every value by 1. This has worked for me because there are no axis or other features visible, so may not be a good solution for more advanced charts.
